queries.sql

```diff
@@ -0,0 +1,10 @@
+
+
+SELECT * FROM animals WHERE name like '%mon';
+SELECT * FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-01-01';
```

Comment:
- [ ] Here we are supposed to select the `name` of all animals born between 2016 and 2019 and not the entire record.
e.g
`SELECT name FROM animals`

NOT ❌ 
`SELECT * FROM animals`
- [ ] Also the date should be between `'2016-01-01' AND '2019-12-31'` and not `'2016-01-01' AND '2019-01-01'`